src/containers/Search.jsx

```diff
@@ -6,7 +6,9 @@ import Map from '../components/Map';
 import {
 	GOOGLE_MAPS_API_KEY,
 	UBER_CLIENT_ID,
-	alternativeCityNamesLookup,
+    alternativeCityNamesLookup,
```

Comment:
indentation, same issue here, check every where else in the changed code @premalrupnur